getting_hired/preparing_for_job_search/project_portfolio.md

```diff
@@ -4,7 +4,7 @@ You don't need a personal website right off the bat but it's good to have and it
 
 For this project, it's best for you to identify the personal websites of several people you admire and see what you think they've done right.  In general, it's best to keep your site simple, clean and to the point rather than bloated and full of "look what I can do!" features.  Let your portfolio of work speak for itself.
 
-* [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/)
+* [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/) 
```

Comment:
It looks like you added white space to the end of this line. Can you please remove it.